Kommunicate-io/Kommunicate-Web-SDK
# FWH-976 To Show chat widget Animation as soon as user sends the message

### What do you want to achieve?
- To Show chat widget Animation as soon as user sends the message
- following cases were updated
1. configured delay < response delay - show message directly
2. configured delay > response delay - show message after (configured delay - response delay)

### PR Checklist


### How was the code tested?

- Make sure the no one is running bot server locally. Else message will sometimes miss
- configure an intent such that it replies with some delay. you can use inline code of webhook or upload a doc to chat gpt
- configure the delay in settings
- test the delay


### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

### Screenshot
- https://github.com/Kommunicate-io/Kommunicate-Web-SDK/assets/121277435/8c32a14a-1742-41d7-9af1-459e54965317




NOTE: Make sure you're comparing your branch with the correct base branch
